integration_test.go

```diff
@@ -157,16 +171,17 @@ func testHelperCartographer(
 	if !utils.SelectContextOrWaitChan(ctx, done) {
 		test.That(t, errors.New("test timeout"), test.ShouldBeNil)
 	}
```

Comment:
Should we wait for `imuDone` here as well?

Reply:
No, since we are currently never passing done to the imuDone channel

Reply:
cool this is a nit but can we move the comment above the `selectContextOrWaitChan`. I think comments typically refer to what is below it